src/Sylius/Bundle/UiBundle/Resources/config/services/template_event.xml

```diff
@@ -26,5 +26,10 @@
             <argument type="service" id="Sylius\Bundle\UiBundle\Registry\TemplateBlockRegistryInterface" />
             <argument type="service" id="Sylius\Bundle\UiBundle\Renderer\TemplateBlockRendererInterface" />
         </service>
+
+        <service id="Sylius\Bundle\UiBundle\Command\DebugTemplateEventCommand">
+            <argument type="service" id="Sylius\Bundle\UiBundle\Registry\TemplateBlockRegistryInterface" />
+            <tag name="console.command" command="sylius:debug:template-event" />
```

Comment:
Why put the command name here and not in the protected static property?

Reply:
No hard feelings on this one - results are the same: https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading.

Reply:
Yes. I am probably more keen to put as much as I can in the code, so it's easier for people to understand it without having to find **both** the command and the service definition to figure out the command to run

Reply:
Makes sense, I'll add it.